ExtraEnemyCustomization/Managers/Properties/Inject/Inject_ProjectileManager_WantToFire.cs

```diff
@@ -0,0 +1,16 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EECustom.Managers.Properties.Inject
+{
+    [HarmonyPatch(typeof(ProjectileManager), nameof(ProjectileManager.WantToFireTargeting))]
+    internal static class Inject_ProjectileManager_WantToFire
+    {
+        internal static void Prefix(ref int burstSize)
+        {
+            burstSize = Inject_EAB_Shooter.LastAgent?.GlobalID ?? 1;
```

Comment:
Null propagation/Null Coalescing doesn't work with Unity's Object types (e.g. Component, Behaviour, MonoBehaviour, GameObject) because Unity overrides the null comparison operator.

[UNT0007: Unity objects should not use null coalescing.](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0007.md)
[UNT0008: Unity objects should not use null propagation.](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0007.md)

Reply:
Didn't knew this was actually a issue.
Will consider adding Unity analyzer